### ISSUE TITLE
[Surgebinding] Radiant Ideals Progression

### ISSUE THREAD

**Issue by leafreynolds** [leafreynolds/cosmere#112]
Once you have attracted a spren and said the first ideal of the knights radiant:
- [ ] #108 

Each order will have their general list of things you need to accomplish before being able to say the next ideal. If you have not met the pre-requisites, `These words are not accepted` message should show.

Second Ideal:
- [ ] Windrunners could need to _successfully_ have defended a village from a raid, not just try.

**Comment by thatvia** [comment]
I could see a pokemon mystery dungeon type thing, when you first make a character, that asks a few questions and gives your character some radiant leaning/influence. Like with protecting a village, the question could ask if you would fight the invaders or evacuate the civilians and give either windrunner or edgedancer progress that way. Sort of a way to give a headstart?

**Comment by thatvia** [comment]
Of course, mystery dungeon also has a little menu at the end of the questions, for if you didn't get the pokemon you were wanting, that lets you pick the one you want to play as.

**Comment by SoaringEaqle** [comment]
Stormfather Bondsmiths should be able to override acceptances of ideals (except for themselves or other bondsmiths), by responding "These words are (not) accepted." in chat within a few seconds.

**Comment by SoaringEaqle** [comment]
Each Ideal could be a mini-goal to go along with one task, making scanning for future Ideals easier
I.e. ideals 
1. First - beating a level I raid
2. Second (defining order) - beating a level II raid while:

- [ ] WR - no villager deaths
- [ ] SB - obtaining some form of legal consent before killing raiders
- [ ] DB - no surges
- [ ] ED - no nitwit deaths
- [ ] TW - protecting a library
- [ ] LW -
- [ ] EC - 
- [ ] WS - building iron golems?
- [ ] SW - 
- [ ] BS - breaking a raiders crossbow and leaving him to join the village

3. Third (clarifying second) - raiding a pillager outpost or a woodland mansion or other structure as applicable while: 

- [ ] WR - 
- [ ] SB - legal consent from a nearby woodland mansion (pillager outpost)
- [ ] DB - 
- [ ] ED - 
- [ ] TW - Finding a library and remaining in it for a period of time before leaving with a number of books (any structure with a library)
      - Leaving the structure with a patchoui book/pamplet. (any structure)
- [ ] LW - 
- [ ] EC - 
- [ ] WS - Freeing allays and Iron Golems 
- [ ] SW - 
- [ ] BS - 

4. Fourth (accepting failure) - Participating in a level V raid or some form of assault on Urithiru while:

- [ ] WR - Villagers die
- [ ] SB - Not obtaining consent
- [ ] DB - Villagers die from surges
- [ ] ED - Nitwits are left out without protection
- [ ] TW - Applying knowledge from 3rd ideal in negative way (i.e practicing hemalurgy)
- [ ] LW -
- [ ] EC - 
- [ ] WS - 
- [ ] SW - Dying and not returning in time
- [ ] BS - 


5. Fifth - Beating the ender dragon, wither, or other dangerous mob while:

- [ ] WR - no Entity death
- [ ] SB - Physically making (instead of obtaining from another source) the legal consent to kill/cause destruction
- [ ] DB - limited destruction
- [ ] ED - 
- [ ] TW - 
- [ ] LW -
- [ ] EC - Using all powers and surges at your disposal.
- [ ] WS - 
- [ ] SW - Not dying
- [ ] BS -
